src/test/components/ActiveTabTimeline.test.js

```diff
@@ -200,34 +202,122 @@ describe('ActiveTabTimeline', function() {
         profile,
         store,
         getResourcesPanelHeader,
+        getResourceFrameTrack,
       };
     }
 
-    it('matches the snapshot of a resources panel', () => {
+    it('matches the snapshot of a resources panel when closed', () => {
       const { container } = setup();
       expect(container.firstChild).toMatchSnapshot();
     });
 
+    it('matches the snapshot of a resources panel when opened', () => {
+      const { container, getResourcesPanelHeader } = setup();
+      fireEvent.click(getResourcesPanelHeader());
+      expect(container.firstChild).toMatchSnapshot();
+    });
+
     it('is closed by default', () => {
-      const { getResourcesPanelHeader } = setup();
-      // TODO: Currently it's not possible to test this without accessing the class
-      // directly but this is not ideal for testing. We should assert user-like actions
-      // instead when we have content in this panel.
-      expect(getResourcesPanelHeader().classList.contains('opened')).toBe(
-        false
-      );
+      const { getResourceFrameTrack } = setup();
+      expect(getResourceFrameTrack()).toBeFalsy();
     });
 
     it('clicking on the header opens the resources panel', () => {
-      const { getResourcesPanelHeader } = setup();
+      const { getResourcesPanelHeader, getResourceFrameTrack } = setup();
       const resourcesPanelHeader = getResourcesPanelHeader();
-      // TODO: Currently it's not possible to test this without accessing the class
-      // directly but this is not ideal for testing. We should assert user-like actions
-      // instead when we have content in this panel.
-      expect(resourcesPanelHeader.classList.contains('opened')).toBe(false);
+      expect(getResourceFrameTrack()).toBeFalsy();
 
       fireEvent.click(resourcesPanelHeader);
-      expect(resourcesPanelHeader.classList.contains('opened')).toBe(true);
+      expect(getResourceFrameTrack()).toBeTruthy();
+    });
+  });
+
+  describe('ActiveTabResourceTrack', function() {
+    function setup() {
+      const { profile, ...pageInfo } = addActiveTabInformationToProfile(
+        getProfileWithNiceTracks()
+      );
+      // Setting the first thread as parent track, the second and third as the iframe tracks.
```

Comment:
Optional: Some ASCII art here to show the relationship would be appreciated. This could be generated with our test utils, or even added as an assertion inline.

Reply:
Put the page page relationship here as a comment. Probably would be good to add the assertion and its function as a followup.